engine/api/action/action.go

```diff
@@ -38,8 +38,8 @@ func InsertAction(tx gorp.SqlExecutor, a *sdk.Action, public bool) error {
 		return sdk.ErrActionLoop
 	}
 
-	query := `INSERT INTO action (name, description, type, enabled, public) VALUES($1, $2, $3, $4, $5) RETURNING id`
-	if err := tx.QueryRow(query, a.Name, a.Description, a.Type, a.Enabled, public).Scan(&a.ID); err != nil {
+	query := `INSERT INTO action (name, description, type, enabled, deprecated, public) VALUES($1, $2, $3, $4, $5, $6) RETURNING id`
+	if err := tx.QueryRow(query, a.Name, a.Description, a.Type, a.Deprecated, a.Enabled, public).Scan(&a.ID); err != nil {
```

Comment:
`INSERT INTO action (name, description, type, enabled, deprecated, public) -->
tx.QueryRow(query, a.Name, a.Description, a.Type, a.Enabled, a.Deprecated, public)

and not 
tx.QueryRow(query, a.Name, a.Description, a.Type, a.Deprecated, a.Enabled, public)